x-pack/plugins/canvas/canvas_plugin_src/functions/common/case.ts

```diff
@@ -34,7 +47,7 @@ export function caseFn() {
   };
 }
 
-async function doesMatch(context, args) {
+async function doesMatch(context: any, args: Arguments) {
```

Comment:
Wondering if `context` is worth reining in, iirc it can't be just anything

Reply:
... also, should not the args of `fn: async (context, args) => {` in line 42 above also be typed?

Reply:
So `case` can literally take anything as a valid context... so `any` in this case is appropriate.  In terms of line 42, `context` and `args` are already strongly typed by the `FunctionSpec` type.
<img width="740" alt="Screen Shot 2019-05-02 at 11 33 52 AM" src="https://user-images.githubusercontent.com/297604/57087520-3b920980-6cce-11e9-9d9b-8ad83833ece9.png">